src/ContentObjectAudit.js

```diff
@@ -25,11 +25,11 @@ const ContentObjectAudit = {
     let audits = [];
     for(response of responses) {
       let url = new URL(response.url);
-      let res = await client.utils.ResponseToJson(response);
       let audit = { host: url.hostname };
       if(!response.ok) {
-        audit.error = res;
+        audit.error = JSON.stringify(response);
```

Comment:
nit: I don't think there's a real reason to stringify the error here, it's fine just putting it in the error field of the audit.

Reply:
If I didn't do this, for some reason the error object doesn't get marshalled; it just returns "Object"